conf/puppet/hiera.yaml

```diff
@@ -13,3 +13,12 @@ hierarchy:
       - "%{::platform_type}.yaml"
       - "%{::component}.yaml"
       - common.yaml
+
+  - name: "AWS Parameter Store"
+    lookup_key: hiera_ssm_paramstore
+    uris:
+      - /
+      - /hiera/%{facts.os.family}/
+    options:
+      region: ap-southeast-2
```

Comment:
Can't have specific region here. There are users using AEM OpenCloud from other regions.

Reply:
What value would you suggest here? is this something we would want to have in the repo as null? If we want it set we would get the value supplied in the conf's via ansible and generate the puppet hiera.yaml instead